src/plugins/intel_npu/src/plugin/npuw/weights_bank.hpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <mutex>
+#include <tuple>
+#include <unordered_map>
+
+#include "openvino/openvino.hpp"
+#include "openvino/runtime/make_tensor.hpp"
+#include "openvino/runtime/tensor.hpp"
+
+namespace ov {
+namespace npuw {
+namespace weights {
+
+class Bank {
+public:
+    Bank(const std::string& device_name);
```

Comment:
`explicit` is probably my most often comment here on Github

Reply:
Reworked